src/components/Button/Button.tsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import { Button as MuiButton, ButtonProps as MuiButtonProps, makeStyles } from '@material-ui/core';
 
-export type ButtonProps = MuiButtonProps;
+export type ButtonProps = MuiButtonProps & {
```

Comment:
Aren't those props available for material buttons by default when we render them as anchors?

Reply:
@KonradSzwarc  if I remember correctly, this was because we do not have `strictNullChecks` set to `true` in `tsconfig`

See:
https://github.com/mui-org/material-ui/issues/22699

Reply:
Ok, adding `strictNullChecks` to our tsconfig looks like a great solution

Reply:
I agree 😄